modules/adding-bare-metal-host-to-cluster-using-yaml.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/managing-bare-metal-hosts.adoc
+
+[id="adding-bare-metal-host-to-cluster-using-yaml_{context}"]
+= Adding a bare metal host to the cluster using YAML in the web console
+
+You can add bare metal hosts to the cluster in the web console using a YAML file that describes the bare metal host.
+
+.Prerequisites
+
+* Install a {op-system} compute machine on bare metal infrastructure for use in the cluster.
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. In the web console, navigate to *Compute* -> *Bare Metal Hosts*.
+. Select *Add Host* -> *New with YAML*.
+. Copy and paste the below YAML, modifying the relevant fields with the details of your host:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: openshift-worker-0-3-bmc-secret
+type: Opaque
+data:
+  username: <secret_username>
+  password: <secret_password>
+---
+apiVersion: metal3.io/v1alpha1
+kind: BareMetalHost
+metadata:
+  name: openshift-worker-0-3
+spec:
+  online: true
+  bmc:
+    address: <bmc_address>
+    credentialsName: openshift-worker-0-3-bmc-secret
+    disableCertificateVerification: True
+    username: <secret_username>
+    password: <secret_password>
+  bootMACAddress: <host_boot_mac_address>
+  hardwareProfile: unknown
+----
+. Select *Create* to save the YAML and create the new bare metal host.
```

Comment:
currently console doesn't support importing multiple YAML files, only one file is allowed. User will see below error if follow steps above
<img width="762" alt="Screen Shot 2021-06-16 at 10 21 31 AM" src="https://user-images.githubusercontent.com/12692381/122148260-c9201b80-ce8c-11eb-8fd1-9fd3bd75426c.png">

Reply:
ok, modified this to not include the `Secret` CR part.

Reply:
It won't work without the Secret though.

Reply:
@aireilly which OCP version is it targetting? in latest OCP 4.8 we added support for multiple YAML